lib/graph3d/Graph3d.js

```diff
@@ -322,18 +327,13 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
 
 
 Graph3d.prototype.getColumnRange = function(data,column) {
-  var minMax;
+  var minMax  = new Range();
 
   for (var i = 0; i < data.length; i++) {
     var item = data[i][column];
-
-    if (i === 0) {
-       minMax = { min: item, max: item};
-    } else {
-      if (minMax.min > item) { minMax.min = item; }
-      if (minMax.max < item) { minMax.max = item; }
-    }
+    minMax.add(item);
```

Comment:
I prefer the term `add()`; this is a question of good commenting/documentation to state the intent.

Wrt. setting/adding: this method `getColumnRange()` deals with single data values. It is not on the level of ranges or coordinates. It is actually the method which initializes a particular range.

Again, this could use some commenting to state the intent.